script/helperFunctions.sh

```
@@ -2793,6 +2793,24 @@ function getRPCUrl() {
   echo "${!RPC_KEY}"
 }
+function getRpcUrlFromNetworksJson() {
+  local NETWORK="$1"
+
+  checkNetworksJsonFilePath || checkFailure $? "retrieve NETWORKS_JSON_FILE_PATH"
+  if [[ ! -f "$NETWORKS_JSON_FILE_PATH" ]]; then
+    echo "Error: JSON file '$NETWORKS_JSON_FILE_PATH' not found." >&2
+    return 1
+  fi
+
+  local RPC_URL=$(jq -r --arg network "$NETWORK" '.[$network].rpcUrl // empty' "$NETWORKS_JSON_FILE_PATH")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)